### PR TITLE
Task#35615

### PR DESCRIPTION
**Pull request by linhpduong**
https://edu-redmine.sun-asterisk.vn/issues/35615
Screen Shot 2021-04-19 at 15 09 11

### CODE REVIEW

**Comment by linhpduong** [comment]
ready

File: src/pug/partials/details/san-pham/list_san-pham/products.pug [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+mixin listProduct
+  article.product
+    .img__container
+        img(src='../assets/images/Product/14-cropped.jpg' alt='')
```
Comment (manhnt-0960) [inline src/pug/partials/details/san-pham/list_san-pham/products.pug:5]: alt

Comment (manhnt-0960) [reply src/pug/partials/details/san-pham/list_san-pham/products.pug:5]: fix vào pull sau

Comment (linhpduong) [reply src/pug/partials/details/san-pham/list_san-pham/products.pug:5]: vang a

**Pull request merged**